src/main/java/menion/android/whereyougo/utils/NotificationService.java

```diff
@@ -87,18 +87,23 @@ private void startNotificationService(boolean background) {
         if (runing && background == foreground) {
```

Comment:
as @Lineflyer wrote:
Just wondering why the variable is `runing` instead of `running`

Reply:
corrected the misspelling